test_main.py

```diff
@@ -0,0 +1,33 @@
+"""
+This is the test !!!
+"""
+import unittest
+from app import get_guess_computer, get_guess_player, mainFunction
+from unittest.mock import patch, MagicMock, Mock
+
+class TestMainCode(unittest.TestCase):
+    
+    def test_get_computer(self):
+            result = get_guess_computer()
+            self.assertGreaterEqual(result, 1)
+            self.assertLessEqual(result, 100)    
+
+    @patch('builtins.input', return_value=22)
+    def test_get_player(self, mock_input):
+            result= get_guess_player()
```

Comment:
quiza seria bueno agegar docstring aca para explicar el test , o cambiar el nombre de funcion

`test_get_player_returns_int`
o
""" Testing that the users input will be returned as int"""